src/structures/user.ts

```diff
@@ -42,7 +42,7 @@ export class User extends Base {
 
   avatarURL(format: ImageFormats = 'webp', size: ImageSize = 512, dynamic: boolean = true): string {
```

Comment:
I don't think dynamic arg is necessary now

Reply:
Better would be to have these arguments as options in an object.

Reply:
deleted